swift_browser_ui_frontend/src/pages/BrowserPage.vue

```diff
@@ -1,12 +1,16 @@
 <template>
   <div id="mainContainer">
-    <div id="subContainer">
+    <nav>
       <BrowserMainNavbar :langs="langs" />
       <BrowserSecondaryNavbar
         :multiple-projects="multipleProjects"
         :projects="projects"
       />
-      <ProgressBar v-if="isUploading || isChunking" />
+    </nav>
+    <div
```

Comment:
should this be `<main>` or maybe `c-main` ?

Reply:
The `div` already has `role="main"` which is similar to `<main>`.
https://github.com/CSCfi/swift-browser-ui/blob/bugfix/accessibility-role-label-issues/swift_browser_ui_frontend/src/pages/BrowserPage.vue#L10-L13

`<c-main>` on the other hand I think it's just the name, I don't think they use `<main>` to represent it.